winrm_plugin/README.md

```diff
@@ -0,0 +1,18 @@
+# cloudify-winrm-plugin
+
+preliminary requirements:
+
+ubuntu side:
+sudo pip install pywinrm 
```

Comment:
When you put pywinrm in the setup.py, you don't need to do pip install. Pip will automatically resolve that when this plugin is installed.